srs_env_model_msgs/PlaneDesc plane
--- 
uint32 object_id